README.md

```diff
@@ -33,7 +33,8 @@ Please note that the [plugin ecosystem in Prettier](https://prettier.io/docs/en/
 
 Plugin for Prettier to format pug code
 
-You can disable code formatting for a particular code block by adding <nobr>`<!-- prettier-ignore -->`</nobr> before ` ```pug `.
+You can disable code formatting for a particular code block by adding <nobr>`<!-- prettier-ignore -->`</nobr> before ` ```pug `,
+as well as by using <nobr>`//- prettier-ignore`</nobr> comments inside your pug templates.
```

Comment:
Could you swap these sentence parts with each other? So `//- prettier-ignore` becomes more priority

Reply:
Yes good point. BTW, the example underneath is wrong, no?

```
<!-- prettier-ignore -->
\`\`\`pug
div.text( color =   "primary",  disabled  ="true"  )
\`\`\`

Prettified code:

\`\`\`pug
.text(color="primary", disabled)
```
\`\`\`

Reply:
Using underscores because I don't know how to nest code in code ;)

Reply:
The point is: The Prettified code shouldn't actually be prettified in the example?

Reply:
No, it's all correct
The `<!-- prettier-ignore -->` only applies to its own "```" block

Reply:
Ouff, sorry! I don't understand what it's evem doing then?

Reply:
I want to present an example of how it would look like to use non-formatting in markdown files

Using it in pug itself should be its own example block :slightly_smiling_face:

Reply:
That's also the reason for the four "\`" instead of three "`"

Reply:
Is this about using it with markdown?

Reply:
yes